docs/src/modules/spring/pages/using-jwts.adoc

```diff
@@ -2,134 +2,7 @@
-
-* Create a new secret suitable for use with the selected algorithm named according to the `--secret` argument.
-* Add a JWT key to the service that references that secret.
-
-The `--issuer` is optional, but recommended. It will ensure that if a JWT specifies an issuer claim (`iss`), only that key will be used to verify that claim. This prevents spoofing of issuer claims. The `--key-id` is required, and should be unique to the service.
-
-The `--secret` argument is optional, if not present, the name of the secret will be the argument passed to `--key-id`.
-
-=== Adding a key for a service
-
-If you've already created a suitable JWT secret, you can add it to a service. For example, perhaps you generated a secret for service A, and you want service B to reuse the same secret. This can be done using the `kalix service jwts add` command:
-
-----
-kalix services jwts add <my-service> \
-  --key-id <my-key-id> \
-  --algorithm HS256 \
-  --issuer <my-issuer> \
-  --secret <my-secret-name>
-----
-
-==== Managing secrets
-
-To create secrets yourself, you can use the `kalix secrets` command. To create a symmetric secret for use with HMAC algorithms:
-
-----
-kalix secrets create symmetric <my-secret-name> \
-  --secret-key-literal "<some-secret-text>"
-----
-
-The secret can also come from a file:
-
-----
-kalix secrets create symmetric <my-secret-name> \
-  --secret-key /path/to/secret.key
-----
-
-To create an asymmetric secret:
-
-----
-kalix secrets create asymmetric <my-secret-name> \
-  --private-key /path/to/private.key \
-  --public-key /path/to/public.key
-----
-
-The public and private keys must be PEM encoded keys, either RSA, ECDSA or Ed25519. We recommend PKCS8 encoded private keys (that is, keys with a PEM header of `BEGIN PRIVATE KEY`) and PKIX encoded public keys (with a header of `BEGIN PUBLIC KEY`), but we also support PKCS1 (`BEGIN RSA PRIVATE/PUBLIC KEY`) and SEC.1 (`BEGIN EC PRIVATE KEY`). For some encodings, `kalix` may prompt to convert them to a format that the JWT support will work with.
-
-The public and private keys are optional, however you must specify at least one of them. For example, if you have a service that only needs the key to validate JWTs issued by another service, you may configure a secret for it that just contains the public key so that it is unable to sign JWTs with that key.
-
-If you only have the private key, you can also ask Kalix to extract the public key from the private key, rather than having to do it manually yourself and passing it using the `--public-key` flag:
-
-----
-kalix secrets create asymmetric <my-secret-name> \
-  --private-key /path/to/private.key \
-  --extract-public-key
-----
+This section describes the practical aspects of configuring JSON Web Tokens (JWTs) with the Java/Scala SDKs, if you are not sure what JWTs are or how they work, see https://docs.kalix.io/security/jwts.html[Understanding JWTs] first. If you already have your JWTs generated and configured for a service
```

Comment:
Missing something in the end of the sentence?

Reply:
Indeed.. Fixed now, thanks 🙇